src/gui/accountstate.cpp

```diff
@@ -247,6 +263,7 @@ void AccountState::setState(State state)
         } else if (_state == Connected && Utility::internetConnectionIsMetered() && ConfigFile().pauseSyncWhenMetered()) {
             _state = PausedDueToMetered;
         }
+        // Do we need an extra state PausedDueToCaptivePortal?
```

Comment:
For now we could reuse "Connecting"? In the end its only to display a different string? In which case we could make the "connecting" message conditional on isBehindCaptivePortal